tests/cql/CqlTypesTest.xml

```diff
@@ -131,12 +131,12 @@
 	</group>
 	<group name="Quantity">
 		<test name="QuantityTest">
-			<expression>150.2 'lbs'</expression>
-			<output>150.2 'lbs'</output>
+			<expression>150.2 '[lb_av]'</expression>
+			<output>150.2 '[lb_av]'</output>
 		</test>
 		<test name="QuantityTest2">
-			<expression>2.5589 'eskimo kisses'</expression>
-			<output>2.5589 'eskimo kisses'</output>
+			<expression>2.5589 '{eskimo kisses}'</expression>
+			<output>2.5589 '{eskimo kisses}'</output>
```

Comment:
`eskimo kisses` is (obviously) not a valid unit (but it _should_ be!).  We need the `{` `}` delimiters to indicate it is a custom unit -- else the `Quantity` is invalid and returned as null.